src/main.js

```diff
@@ -16,6 +16,15 @@ Vue.config.productionTip = false;
 registerModals();
 sync(store, router);
 
+store.watch(
+  ({ isBackendLive }) => isBackendLive,
+  async (isBackendLive) => {
+    const targetRouteName = isBackendLive ? 'tips' : 'maintenance';
+    if (router.currentRoute.name === targetRouteName) return;
+    await router.push({ name: targetRouteName });
+  },
+);
```

Comment:
Don't want to make store and router depend on each other. Can't find a better place for this code 🤔